### ISSUE TITLE
Linux | Wayland icon instead of normal Zen one on taskbar

### ISSUE THREAD

**Issue by sezamn1** [zen-browser/desktop#3578]
### Captchas

- [X] I have read the instructions.
- [x] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

The icon on the taskbar seems to be broken. Earlier versions (like the alpha ones) were acting correctly (the pinned icon was the same one as the open one). Now when pinning Zen to taskbar, and opens it, the icons appear separately.
![obraz](https://github.com/user-attachments/assets/b035df52-170b-49a2-9638-c63e4d49b8a9)
![obraz](https://github.com/user-attachments/assets/bbe1407b-9d7c-4475-8eb6-441c3a19673f)
The wayland icon also has no control options, like opening new windows, compared to the pinned Zen icon
![obraz](https://github.com/user-attachments/assets/c5c4fedb-3c73-4596-985a-3c29150b9f15)
![obraz](https://github.com/user-attachments/assets/46879a42-8bb0-4ffe-a953-7ae480a56f88)

Installation method: AppImage, thru terminal
Desktop: Arch Linux, KDE Plasma 6.2.4

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.2-b.0

### What platform are you seeing the problem on?

Linux

**Comment by CcMenta** [comment]
I also use arch but I use the aur package and it works fine, so it's most likely an appimage issue.

**Comment by NTICompass** [comment]
I use Fedora 41, and I installed [`zen-browser-avx2` via copr](https://copr.fedorainfracloud.org/coprs/sneexy/zen-browser/).  I'm having the same issue.

**Comment by sezamn1** [comment]
> I also use arch but I use the aur package and it works fine, so it's most likely an appimage issue.

Yes, after installing AUR package it seems to be fixed now, thank you. However I still will leave the issue open.

**Comment by sezamn1** [comment]
sorry, im new to github

**Comment by NTICompass** [comment]
This issue seems fixed in 1.0.2-b.1 :-)

**Comment by NTICompass** [comment]
It was fixed, but now it's broken again in 1.6b.

![401999512-07470ab6-5eb0-49de-81fa-3975a4b80a35](https://github.com/user-attachments/assets/407587f8-ec36-4bb5-a025-3433e0aee660)

**Comment by imoize** [comment]
on gnome also have this issue.

**Comment by sezamn1** [comment]
![obraz](https://github.com/user-attachments/assets/ec36d372-3c10-44da-b052-fb36453ea892)
![obraz](https://github.com/user-attachments/assets/1ac69d43-2b40-452f-8cf8-3ad36919b98e)
why, oh why is it happening again?
Also, unrelated but my profile switched a third time now, and i CAN'T switch back
![obraz](https://github.com/user-attachments/assets/0619b2bf-4302-4e2b-8527-b69d62bc5b83)

**Comment by mr-cheffy** [comment]
You need to provide a desktop icon. Alternately, you can install the flatpak or appimage that already have them inserted
